### PR TITLE
Add steinbock recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@jwindhager do you agree with being added as maintainers?

**Comment by jwindhager** [comment]
Hi @goanpeca, steinbock is currently being maintained by @Milad4849

**Comment by Milad4849** [comment]
Hi @goanpeca, you can add me as a maintainer.

**Comment by goanpeca** [comment]
Windows failure is expected. Package is noarch and some win deps are missing.
        
This one is ready for review @conda-forge/staged-recipes :)

**Comment by dopplershift** [comment]
The build error on windows isn't about missing dependencies, it's complaining about `steinbock.exe`. Missing entry point?

**Comment by goanpeca** [comment]
>The build error on windows isn't about missing dependencies, it's complaining about steinbock.exe. Missing entry point?

You are right @dopplershift!

My bad, fixed it!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/steinbock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - scipy
+    - tifffile
+    - xtiff
+
+test:
+  imports:
+    - steinbock
+  commands:
+    - pip list
+    # - steinbock
```
Comment (jaimergp) [inline recipes/steinbock/meta.yaml:46]: Let's have at least something to test that the entry point was created:

```suggestion
    - which steinbock
```

File: recipes/steinbock/yum_requirements.txt
Comment (jaimergp) [inline recipes/steinbock/yum_requirements.txt:1]: Do we need these here?

Comment (goanpeca) [reply recipes/steinbock/yum_requirements.txt:1]: Testing now

Comment (goanpeca) [reply recipes/steinbock/yum_requirements.txt:1]: Removed!, not needed

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/steinbock/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - scipy
+    - tifffile
+    - xtiff
+
+test:
+  imports:
+    - steinbock
+  commands:
+    - pip list
+    - which steinbock
```
Comment (xhochy) [inline recipes/steinbock/meta.yaml:46]: This is failing on Windows.

**Review by xhochy — CHANGES_REQUESTED** [review]

**Pull request merged**